src/components/Footer.css

```diff
@@ -5,7 +5,10 @@ footer {
   font-size: 0.9em;
   line-height: 1.3;
   width: 100%;
+  position: relative;
   margin: 0;
+  bottom: 0;
```

Comment:
as we're currently not using the bottom-alignment, I think this and the position above can be deleted